community/knative/charts/serving/charts/monitoring/charts/prometheus/templates/node-exporter-pod.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: node-exporter
+  namespace: "knative-monitoring"
+  labels: {{ include "labels" . | nindent 4 }}
+    app: node-exporter
+spec:
+  template:
+    metadata:
+      labels: {{ include "labels" . | nindent 8 }}
```

Comment:
This sets spec.template.metadata.labels.chart to "prometheus-0.1.0", which contains the chart version; change this so that the chart version is not present.

Reply:
will be fixed in next commit